src/utils/daos_hdlr.c

```diff
@@ -2628,7 +2357,8 @@ fs_copy_dir(struct cmd_args_s *ap,
 	/* set original source perms on directories after copying */
 	rc = file_chmod(ap, dst_file_dfs, dst_path, src_stat->st_mode);
 	if (rc != 0) {
-		DH_PERROR_SYS(ap, rc, "updating destination permissions failed on '%s'", dst_path);
+		rc = daos_errno2der(rc);
```

Comment:
curious why you are changing return codes to DER from errno?

Reply:
@mchaarawi all of the higher level functions just return DER codes. The file_* wrappers themselves still return errno codes but are converted in calling functions. I suppose the idea was just to do it consistently. It could have returned only errno codes from all higher level functions but the thinking is that since it is a user facing DAOS utility it might be better if they all returned DER codes.

Reply:
@mchaarawi all of the higher level functions just return DER codes. The file_* wrappers themselves still return errno codes but are converted in calling functions. I suppose the idea was just to do it consistently. It could have returned only errno codes from all higher level functions but the thinking is that since it is a user facing DAOS utility it might be better if they all returned DER codes.